games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils cmake-utils gnome2-utils vcs-snapshot user
+
+DESCRIPTION="An InfiniMiner/Minecraft inspired game"
+HOMEPAGE="http://minetest.net/"
+SRC_URI="http://github.com/minetest/minetest/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+curl dedicated doc leveldb luajit nls redis +server +sound +truetype"
```

Comment:
Do I understand correctly that there are three valid USE flag combinations:

* `USE=dedicated` which builds only server,
* `USE="-dedicated server" which builds client+server,
* `USE="-dedicated -server" which builds only client?

If so, I think we should really consider jumping to the cleaner `USE=client` and `USE=server` with `REQUIRED_USE` requiring at least one of them. Otherwise you could do `REQUIRED_USE="dedicated? ( server )"`  to avoid absurd USE combinations but I think client+server is more user-friendly :).